zcash_client_backend/src/fees.rs

```diff
@@ -24,10 +25,10 @@ pub mod zip317;
 /// `ChangeValue` represents either a proposed change output to a shielded pool
 /// (with an optional change memo), or if the "transparent-inputs" feature is
 /// enabled, an ephemeral output to the transparent pool.
-#[derive(Clone, Debug, PartialEq, Eq)]
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
 pub struct ChangeValue(ChangeValueInner);
 
-#[derive(Clone, Debug, PartialEq, Eq)]
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
```

Comment:
I'm surprised that serde serialization is needed for this type. Ordinarily, this type would only be serialized via a `Proposal`, in which case it should use the `protobuf` encoding. What is the reason that this serialization is being added?